cmd/podman/common.go

```diff
@@ -375,10 +376,16 @@ func getCreateFlags(c *cliconfig.PodmanCommand) {
 		"name", "",
 		"Assign a name to the container",
 	)
-	createFlags.String(
-		"net", getDefaultNetwork(),
-		"Connect a container to a network",
-	)
+	aliasNet := func(f *pflag.FlagSet, name string) pflag.NormalizedName {
+		switch name {
+		case "net":
+			name = "network"
+			break
+		}
+		return pflag.NormalizedName(name)
+	}
+
+	createFlags.SetNormalizeFunc(aliasNet)
```

Comment:
I think we already have a version of this in create.go/run.go - can they be combined?